packages/core/src/index.ts

```diff
@@ -217,6 +218,8 @@ function init(options: InitOptions): OnboardAPI {
     unstoppableResolution
   })
 
+  updateAppMetadata(appMetadata)
```

Comment:
I think we need an undefined check here and remove the default null initializer on ln 95

Reply:
updated!